README.md

```diff
@@ -3,34 +3,61 @@
 > A Python wrapper for using the [ocr.space API](https://ocr.space/ocrapi).
 
 ## Installation
+
 Simply install from `pip`:
+
 ```sh
 pip install ocrspace
 ```
 
 ## Use
+
+First, get an API key from [Free OCR API](https://ocr.space/OCRAPI), otherwise the default apikey `helloworld` will be used, which is [severely rate limited.](https://ocr.space/faq#span12)
+
 First you'll need to import and instantiate the API wrapper:
-```py
+
+```python
+import ocrspace
+
+api_key = 'apikey retrieved from Free OCR API'
+
+api = ocrspace.API(api_key=api_key)
+# Or if you have a custom API host, API key or desired language, pass those:
+api = ocrspace.API(endpoint='https://example.host', api_key=api_key, language=ocrspace.Language.Croatian)
+```
+
+If you wish to change the OCR engine used, use the enum class Engine from ocrspace and pass the value of ocrspace.Engine.Engine.ENGINE_2 to the api instantiation. By default it uses ocrspace.Engine.Engine.ENGINE_1:
+
+```python
 import ocrspace
-api = ocrspace.API()
+
+api_key = "apikey retrieved from Free OCR API"
+
+api = ocrspace.API(api_key=api_key, engine=ocrspace.Engine.Engine.ENGINE_2)
 # Or if you have a custom API host, API key or desired language, pass those:
-api = ocrspace.API(endpoint='https://example.host', api_key='Insert key here', language=ocrspace.Language.Croatian)
+api = ocrspace.API(endpoint='https://example.host', api_key=api_key, language=ocrspace.Language.Croatian, engine=ocrspace.Engine.Engine.ENGINE_2)
```

Comment:
And here

Reply:
resolved on c9ffd78ddc93ede49af145fe706ee269c4e0a42c